nextcord/http.py

```diff
@@ -330,7 +333,9 @@ async def request(
                         data = await json_or_text(response)
 
                         # check if we have rate limit header information
-                        remaining = response.headers.get("X-Ratelimit-Remaining")
+                        limit = response.headers.get("X-RateLimit-Limit") or "5"
+                        remaining = response.headers.get("X-Ratelimit-Remaining") or "0"
```

Comment:
```suggestion
                        limit = response.headers.get("X-RateLimit-Limit")
                        remaining = response.headers.get("X-Ratelimit-Remaining")
```

Reply:
Actually hmm, hold this change. I need to look into better handling for global limits

Reply:
Okay cool, inline with my new comment I still think this change is valid.

However this info should be for non global ratelimits only